hw/top_earlgrey/data/ip/chip_hmac_testplan.hjson

```diff
@@ -64,26 +62,37 @@
       stage: V2
       si_stage: SV3
       lc_states: ["PROD"]
-      tests: ["chip_sw_hmac_multistream"]
+      tests: ["chip_sw_hmac_oneshot"]
       bazel: ["//sw/device/tests/crypto:hmac_functest"]
     }
     {
-      name: chip_sw_hmac_stream_mode
+      name: chip_sw_hmac_multistream_mode
       desc: '''Verify HMAC context save/restore functionality.
 
-            - Verify that the HMAC block is able to support context save/restore accross the following
-              modes of operation:
-              - HMAC versus SHA2
-              - Different internal block size, e.g. 32 vs 64 bit for SHA2-256 versus SHA2-384/512.
-            - Ensure that this can be supported by initializing multiple operations and interleaving the
-              HMAC update and final calls.
+            - Verify that HMAC block can stream 6 SHA-2/HMAC operations in parallel:
+              - One test vector for each configuration:
+                - SHA2-256
+                - SHA2-384
+                - SHA2-512
+                - HMAC-SHA2-256
+                - HMAC-SHA2-384
+                - HMAC-SHA2-512
+              - For each vector, message input is split into two chunks, and each
+              operation is completed with 2 update calls.
```

Comment:
In a future refinement in an upcoming PR, can we randomize the number of update calls, e.g., to range between [1, 4] for the different vectors?